includes/amp-helper-functions.php

```diff
@@ -54,6 +54,17 @@ function amp_deactivate( $network_wide = false ) {
  * @since 1.5
  */
 function amp_bootstrap_plugin() {
+	/**
+	 * Filters whether AMP is enabled on the current site.
+	 *
+	 * Useful if the plugin is network activated and you want to turn it off on select sites.
```

Comment:
The purpose of the filter would seem to still be respected with the move to `amp_bootstrap_plugin`. In other words, it doesn't seem that this filter would be applied in the theme.